group1/comm/static/comm_tool.css

```diff
@@ -524,17 +535,23 @@ button#deleteButton {
 /* Begin Video Area Styles */
 
 .message_area{
+  height: 850px;
+  width: 80%;
   display: flex;
   flex-direction: row;
   justify-content: flex-start;
+  overflow-x: hidden;
 }
 
 .videoArea{
   display: flex;
-  flex-direction: row;
-  justify-content: center;
+  flex-direction: column;
+  justify-content: flex-end;
   align-items: center;  
-  width: 100%;
+  align-content: flex-end;
+  width: 45%;
+  height: 570px;
```

Comment:
It's been awhile since I've done much frontend work, but isn't it generally discouraged to mix absolute and relative styling?  I'm asking mostly out of curiosity.  If this is working, we should stick with it.  We don't have time today to go through all this styling and decide if it should all be either relative or absolute or a mix of both really.

Reply:
Yes, it is in fact janky to mix relative and absolute styling.  However, this was the only way I could get it to work in the limited time I had available.  As you said, we can go back and make this work.